app/server.rb

```diff
@@ -996,30 +996,37 @@ def getItemUsage(itemID)
     (item.genre == "dissertation" && pubDate) and pubDate = pubDate.to_s.sub(/-01-01$/, '')
     begin
       body = {
-        :id => shortArk,
-        :citation => citation,
-        :title => citation[:title],
         # ToDo: Normalize author attributes across all components (i.e. 'family' vs. 'lname')
-        :authors => authors,
-        :editors => editors.any? ? editors : nil,
-        :advisors => advisors.any? ? advisors : nil,
-        :published => pubDate,
         :added => item.added,
-        :genre => item.genre,
-        :status => item.status,
-        :rights => item.rights,
-        :content_type => item.content_type,
+        :advisors => advisors.any? ? advisors : nil,
+        :altmetrics_ok => false,
+        :appearsIn => unitIDs ? unitIDs.map { |unitID| {"id" => unitID, "name" => Unit[unitID].name} }
+                              : nil,
+        :attrs => attrs,
+        :authors => authors,
+        :citation => citation,
         :content_html => getItemHtml(item.content_type, id),
         :content_key => calcContentKey(shortArk),
         :content_prefix => content_prefix,
+        :content_type => item.content_type,
+        :data_digest => item.data_digest,            # Strictly used for admin reference
+        :editors => editors.any? ? editors : nil,
+        :genre => item.genre,
+        :id => shortArk,
+        :index_digest => item.index_digest,          # Strictly used for admin reference
+        :last_indexed => item.last_indexed,          # Strictly used for admin reference
+        :oa_policy => item.oa_policy,                # Strictly used for admin reference
+        :ordering_in_sect => item.ordering_in_sect,  # Strictly used for admin reference
         :pdf_url => pdf_url,
-        :attrs => attrs,
+        :published => pubDate,
+        :rights => item.rights,
         :sidebar => unit ? getItemRelatedItems(unit, id) : nil,
-        :appearsIn => unitIDs ? unitIDs.map { |unitID| {"id" => unitID, "name" => Unit[unitID].name} }
-                              : nil,
+        :source => item.source,                      # Strictly used for admin reference
+        :status => item.status,
+        :submitted => item.submitted,                # Strictly used for admin reference
+        :title => citation[:title],
```

Comment:
I've alphabetized these. And also now added data_digest, index_digest, last_indexed, oa_policy, ordering_in_sect, source, and submitted